packages/cli/test/integration/shared/testDb.ts

```diff
@@ -96,7 +96,12 @@ export async function init() {
 
 		try {
 			const schema = config.getEnv('database.postgresdb.schema');
-			await exec(`psql -d ${testDbName} -c "CREATE SCHEMA IF NOT EXISTS ${schema}";`);
+			const exportPasswordCli = pgOptions.password
+				? `export PGPASSWORD=${pgOptions.password} && `
+				: '';
```

Comment:
Thanks for fixing this! This will help when we add a dockerized PG service to the CI checks.